### PR TITLE
Update "nudus" skin and more

### PR DESCRIPTION
**Pull request by akinokonomi**
Any input is very welcome.

EDIT:

Screenshots

rss-dark2
rss-light

### CODE REVIEW

**Comment by martinrotter** [comment]
Is this still WIP or should I merge?

**Comment by akinokonomi** [comment]
Give me a couple of days.

**Comment by martinrotter** [comment]
No problem, just wanted to be sure before merging.

**Comment by martinrotter** [comment]
I made a checkout and testing your changes.

File: resources/rssguard.qrc [hunk 1]
```diff
@@ -81,13 +81,20 @@
...
     <file>skins/vergilius/qt_style.qss</file>
 
-    <file>skins/nudus/html_adblocked.html</file>
-    <file>skins/nudus/html_enclosure_every.html</file>
-    <file>skins/nudus/html_enclosure_image.html</file>
-    <file>skins/nudus/html_single_message.html</file>
-    <file>skins/nudus/html_wrapper.html</file>
-    <file>skins/nudus/metadata.xml</file>
-    <file>skins/nudus/qt_style.qss</file>
+    <file>skins/nudus-base/html_adblocked.html</file>
```
Comment (martinrotter) [inline resources/rssguard.qrc:84]: Nice, seems that you used that "drop-in" file functionality, nice.

File: resources/skins/nudus-dark/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<skin version="0.02" base="nudus-base">
+  <author>
+    <name>akinokonomi</name>
+  </author>
+  <palette>
+    <color key="FgInteresting"          >#85ACF6</color>
```
Comment (martinrotter) [inline resources/skins/nudus-dark/metadata.xml:7]: This is some of the most bizzare XML formatting I've ever seen. Not saying it's good or bad, just unusual.

File: resources/skins/nudus-dark/qt_style.qss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+QTextEdit { /* TODO: Is this used to be what QPlainTextEdit is right now? But highlight works fine there presently. Should be removed? */
+    selection-background-color: yellow;
+}
```
Comment (martinrotter) [inline resources/skins/nudus-dark/qt_style.qss:3]: Can be likely removed.

**Comment by martinrotter** [comment]
Overally, fairly nice job.

I think we can for sure remove "dark" skin afterwards and even maybe "vergillius".

File: resources/skins/nudus-dark/metadata.xml (hunk 2 above)
Comment (akinokonomi) [reply resources/skins/nudus-dark/metadata.xml:7]: Yeah, this is the first time I'm doing such thing, too... But I've seen something similar somewhere before. 

It remains quite readable while looking more consistent. I would not say this is balanced, but it's close to be.
Any suggestions are welcome too.

File: resources/skins/nudus-dark/qt_style.qss [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+/* TODO: Fine for now, may be improved in future */
+QProgressBar {
+    background-color: palette(highlight);
+    color: palette(window);
+}
+
+QStatusBar::item { /* TODO: should this be removed? I don't see any changes in status bar */
+    border: none;
```
Comment (akinokonomi) [inline resources/skins/nudus-dark/qt_style.qss:28]: @martinrotter And what about this?

File: resources/skins/nudus-base/html_style_base.scss [hunk 5]
```diff
@@ -191,7 +181,8 @@ pre {
     background-color: $ccodeblock;
     overflow-x: auto;
     padding: 7px 13px;
-    // For cases of <pre style='white-space:pre-wrap;'>
+    tab-size: 4;
```
Comment (martinrotter) [inline resources/skins/nudus-base/html_style_base.scss:184]: Personally, I like just 2 tabs/spaces, just a comment.

File: resources/skins/nudus-base/html_style_base.scss [hunk 6]
```diff
@@ -211,6 +202,8 @@ pre {
     }
 }
 
+// TODO: Perhaps add kbd
```
Comment (martinrotter) [inline resources/skins/nudus-base/html_style_base.scss:205]: Sure, why not. :)

File: resources/skins/nudus-base/html_style_base.scss (hunk 5 above)
Comment (akinokonomi) [reply resources/skins/nudus-base/html_style_base.scss:184]: Ah, of course you do.

File: resources/skins/nudus-dark/qt_style.qss (hunk 4 above)
Comment (akinokonomi) [reply resources/skins/nudus-dark/qt_style.qss:24]: Any comment?

File: resources/skins/nudus-base/html_style_base.scss (hunk 6 above)
Comment (akinokonomi) [reply resources/skins/nudus-base/html_style_base.scss:205]: Yeah, it just so I won't forget.

I'll properly squash commits afterwards, anyhow.

File: resources/skins/nudus-dark/qt_style.qss (hunk 4 above)
Comment (martinrotter) [reply resources/skins/nudus-dark/qt_style.qss:24]: I remember that it had some offect when other Qt styles were used - some styles made statusbar crazily thick, this css made it a bit smaller.

Comment (akinokonomi) [reply resources/skins/nudus-dark/qt_style.qss:24]: I see. Will test.

Comment (martinrotter) [reply resources/skins/nudus-dark/qt_style.qss:24]: But it was long time ago, I believe it was "CDE" or "gnome" style which was used in Qt 4 era.

https://aur.archlinux.org/packages/qt5-styleplugins/

Comment (akinokonomi) [reply resources/skins/nudus-dark/qt_style.qss:24]: > in Qt 4 era.

Oh. Well, I'll still see if there would be any regressions.

Thank you.

Comment (akinokonomi) [reply resources/skins/nudus-dark/qt_style.qss:24]: Still can see problems with it in the most recent version of `qt5-styleplugins` (2017), with styles such as _motif_, _gtk 2_, _cde_, perhaps other.

Will add a comment explaining it.

**Comment by akinokonomi** [comment]
I think we are there.
If I don't notice any regressions in the next day or two, I'll squash commits by groups, and they are ready to be merged.

**Comment by akinokonomi** [comment]
Ready to be merged. I hope.

**Comment by martinrotter** [comment]
I am I'll, review later...

**Comment by akinokonomi** [comment]
No problem. Get well soon.

**Comment by akinokonomi** [comment]
Reverted the value of table width (back to 100%), and removed colour override for the table.

**Review by martinrotter — APPROVED** [review]
Looks like quality stuff. Will merge, We will sort things as they come.

**Pull request merged**